default/customizations/custom_sitreps.txt

```diff
@@ -0,0 +1,78 @@
+// The UserSitRepLabel tags and data are required if the message template (and label) is simply provided here rather than being drawn from the stringtable.
+// The data value for the UserSitRepLabel serves as the label to be used in the SitReps filters, and CustomSitrepPriority1 through CustomSitrepPriority4
+// will be at the top of the SitReps display.  As seen below, other labels are fine as well, but they cannot have any spaces if you want to use them in the SITREP_PRIORITY_ORDER
+
+effectsgroups = [
+
+
+EffectsGroup
+    scope = Capital
+    activation = Turn high = 1
+    effects =
+        GenerateSitRepMessage
+            message = "This is a sample Custom Sitrep, which is enabled by the [[tech SPY_CUSTOM_ADVISORIES]] and all the busy empire servants reporting information back to %system%"
+            icon = "icons/tech/categories/spy.png"
+            parameters = [
+                tag = "UserSitRepLabel" data = "CustomSitrepPriority1"
```

Comment:
Isn't that though pretty much just essentially moving the 'tag = "UserSitRepLabel" data = "CustomSitrepPriority1"' a few lines over, and maybe using different words than tag and data?
It rather seems like a "6 of one, half a dozen of the other" kind of thing.  You have a firm view of what the parameters are for based on current usage, but I think to anyone else these meet the general qualifications of a parameter for the message.